packages/backend/src/tools/assert.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line
+export function assert(condition: any, message?: string): asserts condition {
```

Comment:
If you used `condition: unknown` you wouldn't have to disable eslint.

Reply:
Yeah i thought about this but I am not sure if this won't break type inference somewhere.. I will check it out.